numpyro/distributions/continuous.py

```diff
@@ -2438,3 +2439,99 @@ def cdf(self, value):
 
     def icdf(self, value):
         return self._ald.icdf(value)
+
+
+class ZeroSumNormal(TransformedDistribution):
+    r"""
+    Zero Sum Normal distribution adapted from PyMC [1] as described in [2,3]. This is a Normal distribution where one or
```

Comment:
@AlexAndorra @aseyboldt @ricardoV94 hey folks, this PR is nearing ready to go. Thank you for all of your help and please let me know if there's more I can add to properly credit all of you and pymc. I've found the ZeroSumNormal to be such a useful tool, cant thank you all enough!

Reply:
Looks good to me. Glad to hear you found it useful :-)